apps/tup-ui/src/pages/Systems/Systems.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { PageLayout, RequireAuth, SystemNavBar } from '@tacc/tup-components';
+import { SystemDetail } from '..';
+import { SectionHeader } from '@tacc/core-components';
+import styles from './Systems.module.css';
+import { useParams } from 'react-router-dom';
+import { useSystemQueue } from '@tacc/tup-hooks';
+
+const Layout: React.FC = () => {
+  // To get the System name for the Page Header
+  let { tas_name } = useParams<{ tas_name: string }>();
+  if (!tas_name) {
+    tas_name = 'frontera';
+  }
```

Comment:
**Maintenance Issue.** This logic already exists ([one instance](https://github.com/TACC/tup-ui/blob/418fd93/apps/tup-ui/src/pages/SystemDetail/SystemDetail.tsx#L7-L10)). How can you develop this to avoid forgetting to update "all the spots" if default system changes?